unittests/tools/test_edgescan_parser.py

```diff
@@ -0,0 +1,64 @@
+from django.test import TestCase
+
+from dojo.tools.edgescan.parser import EdgescanParser
+from dojo.models import Test
+
+
+class TestEdgescanParser(TestCase):
+
+    def test_parse_file_with_no_vuln_has_no_findings(self):
+        with open("unittests/scans/edgescan/no_vuln.json") as testfile:
+            parser = EdgescanParser()
+            findings = parser.get_findings(testfile, Test())
+            self.assertEqual(0, len(findings))
+
+    def test_parse_file_with_one_vuln_has_one_findings(self):
+        with open("unittests/scans/edgescan/one_vuln.json") as testfile:
+            parser = EdgescanParser()
+            findings = parser.get_findings(testfile, Test())
+            self.assertEqual(1, len(findings))
+            finding = findings[0]
+            self.assertEqual(finding.title, "Cross-site scripting (reflected)")
+            self.assertEqual(finding.date, "2014-12-05")
+            self.assertEqual(finding.cwe, 75)
+            self.assertEqual(finding.cve, "CVE-2021-5300")
+            self.assertEqual(finding.cvssv3, "AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:L/A:N")
```

Comment:
The CVSS vector format is not good. The field need the complete form with the prefix (like `CVSS:3.0/AV:A/AC:H/PR:L/UI:R/S:C/C:N/I:L/A:N` **not** `AV:A/AC:H/PR:L/UI:R/S:C/C:N/I:L/A:N`)

Reply:
Fixed in d377a80.